test/lint/lint-files.py

```diff
@@ -145,26 +162,26 @@ def check_all_file_permissions() -> int:
             continue
         else:
             print(
```

Comment:
Could put `assert(0)` here as well, as this is guaranteed to never happen. Git only stores an 'executable' bit, and its expanded permissions will always be 644 or 755.